### ISSUE TITLE
Feature: Pan graph to center

### ISSUE THREAD

**Issue by hastebrot** [dagrejs/dagre-d3#98]
A function to pan the whole graph to the center of `<svg>` would be nice. Or even a flag that auto-recenters the graph on every change.

Background: I thought of some improvements to the interactive demo (based on the latest release). It would have the `<svg>` element in the center of the page and atop a textarea with the graphviz definition which is initially very small (less than five lines height). It will automatically resize if the available space is too low. Or even add a "simple mode" that initially contains only a one liner (e.g. `foo -> bar; foo -> baz`) and adds this content between `digraph {`and `}`.

**Comment by Qix-** [comment]
Yes please, the fact the graph starts in the top left is aesthetically not the greatest.
